themes/default/content/docs/guides/self-hosted/_index.md

```diff
@@ -8,7 +8,9 @@ meta_desc: Pulumi Enterprise Edition gives you the option to self-host Pulumi wi
 ---
 
 {{% notes type="info" %}}
-Self-hosting is only available with the **Pulumi Enterprise Edition**. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.
+This page is for Self-Hosting of the **Pulumi Enterprise Edition**. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.
```

Comment:
I believe the original sentence here is a bit more clear and succinct (but with a slight tweak).

```suggestion
Self-hosting is only available with **Pulumi Enterprise Edition**. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.
```